README.md

```diff
@@ -6,7 +6,7 @@ Command line utility for creating new releases of libuv.
 
 1. Make sure you have the most up-to-date version of this tool and the libuv
 branch that will be used to create the release.
-2. Currently, the release tool requires Node.js v0.10.x. You may want to use
+2. Currently, the release tool requires Node.js v12.10.x. You may want to use
```

Comment:
Really excited to see this! Are the changes in this PR enough to make it work with Node 12?

Reply:
I don't really know, but I know it won't work in v0.10.x, since I used an API that is available starting in v12